### ISSUE TITLE
Choose merge method when auto-merging

### ISSUE THREAD

**Issue by coreywagehoft** [jpmcb/prow-github-actions#29]
**Is your feature request related to a problem? Please describe.**
The default strategy for merging pull requests when using `/lgtm` is a pull request merge.  Some users might want to flexibility to control the merge strategy.

I took a stab at trying to implement this, but I have not worked with TypeScript, and could not easily.

**Describe the solution you'd like**
A clear and concise description of what you'd like to happen.
I would like to be able to choose the merge strategy when creating the action workflow for the LGTM cron job.  Here is an example of what the action workflow could look like

```yaml
name: "Merge on lgtm label"
on:
  schedule:
  - cron: "0 * * * *"

jobs:
  execute:
    runs-on: ubuntu-latest
    steps:
      - uses: jpmcb/prow-github-actions
        with:
          jobs: 'lgtm'
          merge-method: squash
          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

**Describe alternatives you've considered**
In my use case we prefer to squash when merging a pull request.  If we can't control this we would need to disable the `/lgtm` command

**Additional context**

Based on your code here: https://github.com/jpmcb/prow-github-actions/blob/main/src/cronJobs/lgtm.ts#L115 there is an option in octokit/rest to set `merge_method` as documented here: https://octokit.github.io/rest.js/v18#pulls-merge

**Comment by jpmcb** [comment]
Hi there! Thanks for opening this issue - yes I think this is a great idea. And yes, looks like there would be a good path forward with the `octokit/rest.js` library to make this configurable. Thanks for linking to the rest documentation!

/priority high

**Comment by jpmcb** [comment]
@coreywagehoft this was addressed with the release of v1.1.0, thanks so much for opening this issue!! Let me know if I can be of any more help
